### PR TITLE
MODEXPW-350 Authority control: create basic infrustructure for new jobs

### PR DESCRIPTION
**Pull request by Shans-Kaluhin**
[Authority control: create basic infrustructure for new jobs](https://issues.folio.org/browse/MODEXPW-350)

### CODE REVIEW

**Comment by Shans-Kaluhin** [comment]
Waits for schemas merge: https://github.com/folio-org/folio-export-common/pull/47

**Review by hasanepam — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -369,7 +372,7 @@
                             <generateApiDocumentation>false</generateApiDocumentation>
                             <generateModels>true</generateModels>
                             <modelsToGenerate>
-                              JobStatus,ExportTypeSpecificParameters,EHoldingsExportConfig,IdentifierType,EntityType,Progress,ExportType,Job,Configuration,ConfigurationCollection,LoggedObjectType,ActionType,LogRecord,logRecord_items,logRecord_linkToIds,LogRecordCollection
```
Comment (viacheslavkol) [inline pom.xml:372]: Why is this removed?

File: src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java [hunk 2]
```diff
@@ -216,6 +219,20 @@ private Job createJobExecutionUpdate(String jobId, JobExecution jobExecution) {
...
+        result.setFileNames(singletonList(fileName));
+      }
+    }
+
+    if (jobExecution.getJobInstance().getJobName().contains(FAILED_LINKED_BIB_UPDATES.getValue())) {
+      String fileName = ExecutionContextUtils.getFromJobExecutionContext(jobExecution, AUTHORITY_CONTROL_FILE_NAME);
+      if (StringUtils.isNotBlank(fileName)) {
+        result.setFileNames(singletonList(fileName));
+      }
+    }
```
Comment (viacheslavkol) [inline src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java:234]: Could be simplified to one `if` statement, also `jobExecution.getJobInstance().getJobName()` could be moved to a variable.

File: pom.xml (hunk 1 above)
Comment (Shans-Kaluhin) [reply pom.xml:372]: We have duplication 'EHoldingsExportConfig' in bursar-export-worker.
All ExportConfig schemas generates in bursar. We should split them in future

File: src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java (hunk 2 above)
Comment (Shans-Kaluhin) [reply src/main/java/org/folio/dew/batch/JobCompletionNotificationListener.java]: Thanks, done

**Review by viacheslavkol — APPROVED** [review]
